### PR TITLE
[WIP] Add pma login demo

### PR DESCRIPTION
**Pull request by glendaviesnz**
#### Changes proposed in this Pull Request

see D33970-code

### CODE REVIEW

File: client/my-sites/hosting/main.js [hunk 1]
```diff
@@ -78,7 +98,10 @@ const Hosting = ( { translate } ) => {
 								'Manage your databases with PHPMyAdmin and run a wide range of operations with MySQL.'
 							) }
 						</p>
-						<Button>{ translate( 'Access PHPMyAdmin' ) }</Button>
+						<Button onClick={ handlePMALogin } className="hosting__card-phpmyadmin-login">
+							{ loadingPMA && <Spinner /> }
+							{ ! loadingPMA && translate( 'Access PhpMyAdmin' ) }
```
Comment (a8ci18n) [inline client/my-sites/hosting/main.js]: :ok: This change is safe, we'll keep the existing 2 translations.

File: client/my-sites/hosting/main.js [hunk 2]
```diff
@@ -35,14 +37,32 @@ const DataSection = ( { title, data } ) => {
...
 		[ translate( 'Password' ) ]: 'test',
 	};
 
+	function handlePMALogin() {
+		setLoadingPMA( true );
+
+		// If we don't want the pma url to pass through the redux store we can just do a raw wpcom.req here
+		// TODO: If we proceed with this it needs to be a POST, but setting apiNamespace is not currently working for POST
+		wpcom.req
+			.get( `/sites/${ siteId }/hosting/pma`, {
```
Comment (gwwar) [inline client/my-sites/hosting/main.js]: If folks decide to go with an API here, we can update this to at least dispatch an HTTP action and use the data-layer. (Try searching for usages of `apiNamespace` under the `client/state/data-layer` folder.

Comment (glendaviesnz) [reply client/my-sites/hosting/main.js]: yeh, it is weird, there are lots of examples of wpcom/v2 being used for a post, but doesn't work when I try it either in requestHttpData or wpcom.req.post - will be something simple I am overlooking for sure.

Do you know off the top of your head of an example/pattern for making a single http request and getting the response from within a component method, as apposed to injecting it into the component with a 'connect' wrapper?

Comment (jeraldjuice) [reply client/my-sites/hosting/main.js]: oh, yup - you need to add an empty body param or the request reverts to using v1.1 for some reason.

Comment (glendaviesnz) [reply client/my-sites/hosting/main.js]: Doh! I get the use of requestHttpData now so no need to look for that 'example/pattern for making a single http request and getting the response from within a component method' @gwwar 

https://github.com/Automattic/wp-calypso/pull/36442/files#diff-33f4ec9b84efb2e6e29fcf410a637a40R25

Thanks @jeraldjuice for showing the proper use of this - I had mapping of state to props stuck in my head with this being being a data layer thing, but it definitely isn't a map state to props thing - https://github.com/Automattic/wp-calypso/pull/36442/files#r335114188

**Comment by glendaviesnz** [comment]
see https://github.com/Automattic/wp-calypso/pull/36907

**Pull request closed without merging**